internal/ansible/runner/internal/inputdir/inputdir.go

```diff
@@ -131,6 +132,18 @@ func (i *InputDir) Write() error {
 		return err
 	}
 
+	if strings.HasPrefix(i.CmdLine, string("'")) && i.CmdLine[0] == i.CmdLine[len(i.CmdLine)-1] {
+		i.CmdLine = i.CmdLine[1 : len(i.CmdLine)-1]
+	}
+
+	cmdLineBytes := []byte(i.CmdLine)
+	if len(cmdLineBytes) > 0 {
+		err = i.addFile("env/cmdline", cmdLineBytes)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
Regards the `addFile`: 
- IMO I do not think that we need to add this func. See that we can replace that by: 

```go
	err := ioutil.WriteFile(fullPath := filepath.Join(i.Path, "path"), content, 0644)
	if err != nil {
		log.Error(err, "Unable to write file", "Path", fullPath)
	}
``` 

- Also, the `log.Error(err, "Unable to write file", "Path", fullPath)` we can replace by: 
```
log.Errorf("Unable to write file (%s) : (%s)", fullPath, err)
````

- The `0644` needs be replaced by the const in the projectutils. See: https://github.com/operator-framework/operator-sdk/blob/master/internal/util/projutil/project_util.go#L31

Reply:
The reason here is because the addFile just do the above code so has no need for an extra func and then, it makes easier understand that it is about to write a file in the disk.

Reply:
All the parameters above (Parameters, EnvVars, Settings) all use the addFile while having their bytes written to disk, so I thought we could use that function instead of repeating the code.

Reply:
Hi @VenkatRamaraju, 

Thank you for your reply. Let's see the addFile: 

/https://github.com/operator-framework/operator-sdk/blob/8303ef95a6503e5da8df2eed5b31f70cf885f6db/internal/ansible/runner/internal/inputdir/inputdir.go#L55-L63

Is not it the same of call the `WriteFile`?. So, why we need that? 

```go
        err := ioutil.WriteFile(fullPath := filepath.Join(i.Path, "path"), content, 0644)
	if err != nil {
		log.Error(err, "Unable to write file", "Path", fullPath)
	}
```
`

Reply:
@camilamacedo86 since this is just reusing the function the rest of the code is already using I think dropping the `addFile` function would make sense to do separately, I don't think it's worth blocking the PR to fix it here

Reply:
Yep .. I just saw that.